### PR TITLE
hosted update on homepage and mark page

### PR DESCRIPTION
**Pull request by krishnaglodha**
I have made changes in the `Index` and `Mark` pages

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
 description: >
-  Build Jekyll site with the GitBook style.
+  Style cookbook for Geostyler.
 
 version:          1.0
 gitbook_version:  3.2.3
 
-url:              'https://sighingnow.github.io'
-baseurl:          '/jekyll-gitbook'
+url:              'https://github.com/geostyler/geostyler-cookbook'
```
Comment (jansule) [inline _config.yml:12]: Non-blocking: We might have to change this url to a github.io url

File: _pages/geometry_type/Mark.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+# **Symbolizer Type**
+
+`Mark` represents the visualization of `Point` data.
+
+Point is a simple geometry type, even though it has position as a coordinates attached to it, we can certainly style it using various ways using  [Geostyler](https://geostyler.org/) 
+
+# **UI of styler**
+
+Once data is loaded, [Geostyler](https://geostyler.org/) automatically detects the geometry type and try to set it up, click in column of *paint bucket*
```
Comment (jansule) [inline _pages/geometry_type/Mark.md]:
```suggestion
Once data is loaded, [Geostyler](https://geostyler.org/) automatically detects the geometry type and tries to set it up, click in column of *paint bucket*

```

File: _pages/geometry_type/Mark.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+
+`Mark` supports various properties to create perfect styling for Points, for each change in the properties, you can see preview
```
Comment (jansule) [inline _pages/geometry_type/Mark.md]:
```suggestion
`Mark` supports various properties to create perfect styling for Points. For each change in the properties, you can see a preview.

```

File: _pages/geometry_type/Mark.md [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+![Mark Symbolizer]( ../../images/mark_sym.png "Mark Symbolizer")
+
+`Mark` supports various properties to create perfect styling for Points, for each change in the properties, you can see preview
```
Comment (jansule) [inline _pages/geometry_type/Mark.md]: Duplicated sentence?

File: _pages/geometry_type/Mark.md [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+`Mark` supports various properties to create perfect styling for Points, for each change in the properties, you can see preview 
+![Mark preview]( ../../images/preview_mark.png "Mark preview")
+
+
+
+## Symbols
+
+`Mark` supports various symbol lists right out of the box such as `circle`,`square`,`triangle`,`star`,`cross`,`x` , it also provides shapes such as `backslash` , `carrow`, `dot`, `horline`, `oarrow`, `plus`, `slash`, `times`, `vertline`.
+
+Once you select these, you can check preview to see how it looks
```
Comment (jansule) [inline _pages/geometry_type/Mark.md]:
```suggestion
Once you select these, you can check the preview to see how it looks

```

File: _pages/geometry_type/Mark.md [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+
+![Mark cross]( ../../images/mark_cross.png "Mark cross")
+
+![Mark arrow]( ../../images/mark_arrow.png "Mark arrow")
+
+
+## Radius
+
+`Radius` is the general term use to define the size of the `Mark` , you can set it up as per convenience
```
Comment (jansule) [inline _pages/geometry_type/Mark.md]:
```suggestion
`Radius` is the general term used to define the size of the `Mark` , you can set it up as per convenience.

```

File: _pages/geometry_type/Mark.md [hunk 7]
```diff
@@ -0,0 +1,88 @@
...
+
+## Fill-color
+
+`Fill-color` allows user to select colour for filling the `Mark` . You can select colour by following methods
```
Comment (jansule) [inline _pages/geometry_type/Mark.md]:
```suggestion
`Fill-color` allows users to select color for filling the `Mark` . You can select a color by following methods:

```

File: _pages/geometry_type/Mark.md [hunk 8]
```diff
@@ -0,0 +1,88 @@
...
+- Selection from Colour Panel
+- Hex ID
+- RGB
+- Selection from predefined colors
+
+![Mark fill_color]( ../../images/mark_fill_color.png "Mark fill_color")
+
+## Opacity
+
+`Opacity` defines the transparency of the `Mark`, values can be set between **0 - 1**, where 0 means transparent and 1 means opaque
```
Comment (jansule) [inline _pages/geometry_type/Mark.md]:
```suggestion
`Opacity` defines the transparency of the `Mark`. Values can be set between **0 - 1**, where 0 means transparent and 1 means opaque.

```

File: _pages/geometry_type/Mark.md [hunk 9]
```diff
@@ -0,0 +1,88 @@
...
+
+## Radius
+
+`Radius` is the general term use to define the size of the `Mark` , you can set it up as per convenience
+
+## Fill-color
+
+`Fill-color` allows user to select colour for filling the `Mark` . You can select colour by following methods
+
+- Selection from Colour Panel
```
Comment (jansule) [inline _pages/geometry_type/Mark.md]:
```suggestion
- Selection from Color Panel

```

File: _pages/geometry_type/Mark.md [hunk 10]
```diff
@@ -0,0 +1,88 @@
...
+## Opacity
+
+`Opacity` defines the transparency of the `Mark`, values can be set between **0 - 1**, where 0 means transparent and 1 means opaque
+
+![Mark opacity]( ../../images/mark_opacity.png "Mark opacity")
+
+
+## Fill-Opacity
+
+Using `Fill-opacity` user can specify the transparency of colour used in `Fill-color` , User can create combination of `Fill-opacity` and `stroke-opacity`
```
Comment (jansule) [inline _pages/geometry_type/Mark.md]:
```suggestion
Using `Fill-opacity`, users can specify the transparency of the color used in `Fill-color`. Users can create a combination of `Fill-opacity` and `Stroke-opacity`

```

File: _pages/geometry_type/Mark.md [hunk 11]
```diff
@@ -0,0 +1,88 @@
...
+
+<aside>
+🚀 Note : This might not reflect properly in preview, but developers are working on it, and will be resolved soon
+
+</aside>
+
+## Stroke-Colour
```
Comment (jansule) [inline _pages/geometry_type/Mark.md]:
```suggestion
## Stroke-Color

```

File: _pages/geometry_type/Mark.md [hunk 12]
```diff
@@ -0,0 +1,88 @@
...
+
+This property defines the color of the border of the `Mark`. Selection of Colour works the same way as `Fill-color`
```
Comment (jansule) [inline _pages/geometry_type/Mark.md]:
```suggestion
This property defines the color of the border of the `Mark`. Selection of color works the same way as `Fill-color`.

```

File: _pages/geometry_type/Mark.md [hunk 13]
```diff
@@ -0,0 +1,88 @@
...
+
+## Stroke-Opacity
+
+This property is used to define transparency of the border. User can set different opacity for `fill` and `stroke` .
```
Comment (jansule) [inline _pages/geometry_type/Mark.md]:
```suggestion
This property is used to define transparency of the border. Users can set different opacity for `fill` and `stroke` .

```

File: _pages/geometry_type/Mark.md [hunk 14]
```diff
@@ -0,0 +1,88 @@
...
+
+Values can be set between **0 - 1**, where 0 means transparent and 1 means opaque
+
+
+## Stroke-Width
+
+`Stroke-Width` is used to set thickness of the border
```
Comment (jansule) [inline _pages/geometry_type/Mark.md]:
```suggestion
`Stroke-Width` is used to set thickness of the border.

```

File: _pages/geometry_type/Mark.md [hunk 15]
```diff
@@ -0,0 +1,88 @@
...
+
+## Rotation
+
+Using `Rotation`, user can rotate the symbol at given value in degree
```
Comment (jansule) [inline _pages/geometry_type/Mark.md]:
```suggestion
Using `Rotation`, users can rotate the symbol at given value in degrees.
```

File: index.md [hunk 16]
```diff
@@ -1,148 +1,49 @@
 ---
 layout: home
 ---
+# 🖥️ Demo
 
-Make Jelly site have a GitBook look!
+All written functions can be tested here Geostyler [Official demo](https://geostyler.github.io/geostyler-demo/) UI. Also checkout [repo](https://github.com/geostyler/geostyler-demo) for the same.
```
Comment (jansule) [inline index.md]:
```suggestion
All written functions can be tested on the [GeoStyler Demo](https://geostyler.github.io/geostyler-demo/). Also checkout the [geostyler-demo repo](https://github.com/geostyler/geostyler-demo) for the same.
```

File: index.md [hunk 17]
```diff
@@ -1,148 +1,49 @@
...
 
-## Demo
+# 🎨 Getting familiar with UI
```
Comment (jansule) [inline index.md]:
```suggestion
# 🎨 Getting familiar with the UI
```

File: index.md [hunk 18]
```diff
@@ -1,148 +1,49 @@
...
 
-Live demo on Github Pages: [https://sighingnow.github.io/jekyll-gitbook](https://sighingnow.github.io/jekyll-gitbook)
+[Geostyler](https://geostyler.org/) [Official demo](https://geostyler.github.io/geostyler-demo/) is a very easy to use, interactive web app which allows user to start testing capabilities of the application without installing it.
```
Comment (jansule) [inline index.md]:
```suggestion
[Geostyler Demo](https://geostyler.github.io/geostyler-demo/) is a very easy to use, interactive web app which allows users to start testing capabilities of the [GeoStyler library](https://github.com/geostyler/geostyler) without installing it.
```

File: index.md [hunk 19]
```diff
@@ -1,148 +1,49 @@
...
-is building HTML files locally and then push to Github repository, usually to the `gh-pages`
-branch. It's quite annoying to repeat such workload and make it hard for people do version
-control via git for when there are generated HTML files to be staged in and out.
+## 1. Language
 
-This theme takes style definition out of generated GitBook site and provided the template
-for Jekyll to rendering markdown documents to HTML, thus the whole site can be deployed
-to [Github Pages][1] without generating and uploading HTML bundle every time when there are
-changes to the original repo.
+[Geostyler](https://geostyler.org/) demo UI can be translated in 4 different languages
```
Comment (jansule) [inline index.md]: Soon to be 5 :smile:. See https://github.com/geostyler/geostyler/pull/1731

File: index.md [hunk 20]
```diff
@@ -1,148 +1,49 @@
...
-for Jekyll to rendering markdown documents to HTML, thus the whole site can be deployed
-to [Github Pages][1] without generating and uploading HTML bundle every time when there are
-changes to the original repo.
+[Geostyler](https://geostyler.org/) demo UI can be translated in 4 different languages 
 
-## How to Get Started
+- EN - English
+- DE - German
+- ES - Spanish
+- 中文 - mandarin
```
Comment (jansule) [inline index.md]:
```suggestion
- 中文 - Mandarin
```

File: index.md [hunk 21]
```diff
@@ -1,148 +1,49 @@
...
+### Expanded mode
 
-The search functionality in jekyll-gitbook theme is powered by the [gitbook-plugin-search-pro][5] plugin and is enabled by default.
+![expand]( images/expanded.png "expand")
 
-[https://sighingnow.github.io/jekyll-gitbook/?q=generated](https://sighingnow.github.io/jekyll-gitbook/?q=generated)
+## 3. Symbolizer Renderer
 
-## Code highlight
+Currently [Geostyler](https://geostyler.org/) supports `Openlayers` and `SLD` rending for the symbol. Changing this setting will reflect the way symbol is rendered in the `Graphical Editor`
```
Comment (jansule) [inline index.md]:
```suggestion
Currently, [Geostyler](https://geostyler.org/) supports `Openlayers` and `SLD` rendering of symbols. Changing this setting will reflect the way symbols are rendered in the `Graphical Editor`
```

File: index.md [hunk 22]
```diff
@@ -1,148 +1,49 @@
...
-[2]: https://pages.github.com/themes
-[3]: https://github.com/sighingnow/jekyll-gitbook/fork
-[4]: https://github.com/allejo/jekyll-toc
-[5]: https://github.com/gitbook-plugins/gitbook-plugin-search-pro
-[6]: https://github.com/rouge-ruby/rouge/tree/master/lib/rouge/themes
-[7]: https://analytics.google.com/analytics/web/
-[8]: https://www.cnzz.com/
-[9]: https://docs.microsoft.com/en-us/azure/azure-monitor/app/app-insights-overview
-[10]: https://github.com/sighingnow/jekyll-gitbook/blob/master/gitbook/custom.css
+Once loaded users can make changes if required or visualize it as it is.
```
Comment (jansule) [inline index.md]:
```suggestion
Once loaded, users can make changes if required or visualize it as it is. 
```

**Comment by jansule** [comment]
Nice work @krishnaglodha! Just fixed some typos.

We can talk about some contents in detail tomorrow.

File: _config.yml (hunk 1 above)
Comment (krishnaglodha) [reply _config.yml:12]: we'll change it later?

File: index.md (hunk 19 above)
Comment (krishnaglodha) [reply index.md]: done I have added

File: _pages/geometry_type/Mark.md (hunk 4 above)
Comment (krishnaglodha) [reply _pages/geometry_type/Mark.md]: removed it

File: _config.yml (hunk 1 above)
Comment (jansule) [reply _config.yml:12]: Yes

**Review by jansule — APPROVED** [review]
Very nice work @krishnaglodha!

**Pull request merged**